meta/template/jit.profile.lua

```diff
@@ -0,0 +1,20 @@
+---#if not JIT then DISABLE() end
+---@meta jit.profile
+
+local profile = {}
+
+---@param mode string
+---@param func fun(L:thread,samples:integer,vmst:string)
```

Comment:
Can you insert some spaces into your `fun()` annotations in this file?
```lua
---@param func fun(L: thread, samples: integer, vmst: string)
```

Reply:
This feature is not supported by formater. You can contribute these changes into the pull.Thanks